utils/build-script-impl

```diff
@@ -2628,10 +2628,32 @@ for host in "${ALL_HOSTS[@]}"; do
                     -DCMAKE_INSTALL_LIBDIR:PATH="lib"
 
                     -DENABLE_SWIFT=YES
-                    -DSWIFT_RUNTIME_LIBDIR:PATH="${SWIFT_BUILD_PATH}/lib/swift/${SWIFT_HOST_VARIANT}/${SWIFT_HOST_VARIANT_ARCH}"
 
                     -DENABLE_TESTING=YES
                   )
+
+                  case ${product} in
+                    libdispatch)
+                      cmake_options=(
+                        ${cmake_options[@]}
+                        -DSWIFT_RUNTIME_LIBDIR:PATH="${SWIFT_BUILD_PATH}/lib/swift/${SWIFT_HOST_VARIANT}/${SWIFT_HOST_VARIANT_ARCH}"
+                        -DBUILD_SHARED_LIBS=YES
+                      )
+                    ;;
+                    libdispatch_static)
+                      cmake_options=(
+                        ${cmake_options[@]}
+                        -DBUILD_SHARED_LIBS=NO
+                        -DSWIFT_RUNTIME_LIBDIR:PATH="${SWIFT_BUILD_PATH}/lib/swift_static/${SWIFT_HOST_VARIANT}/${SWIFT_HOST_VARIANT_ARCH}"
```

Comment:
This is a shortcoming in the libdispatch build.  https://github.com/apple/swift-corelibs-libdispatch/pull/397 should address that.  This shouldn't be set.

Reply:
Dropped